tap_jira/streams.py

```diff
@@ -4,11 +4,48 @@
 import dateparser
 
 from singer import metrics, utils, metadata, Transformer
+from singer.transform import SchemaMismatch
+from dateutil.parser._parser import ParserError
 from .http import Paginator,JiraNotFoundError
 from .context import Context
 
 DEFAULT_PAGE_SIZE = 50
 
+def handle_date_time_schema_miss_match(exception, record): # pylint: disable=inconsistent-return-statements
+    """
+    Handling exception for date-time value out of range.
+    """
+
+    if ("{'format': 'date-time', 'type': ['string', 'null']}"
```

Comment:
Each time we will get String first then null? or are there any chances where we might get null first then string?

Reply:
We've verified and observed that we're getting string first and then null for the type in the error message.

Reply:
For each stream?

Reply:
If we have a different structure then we must modify that in the catalog.

Reply:
Yes, we are getting the same error for each stream.

Reply:
ok